src/main/java/com/spikes2212/command/genericsubsystem/GenericSubsystem.java

```diff
@@ -21,11 +21,11 @@ public abstract class GenericSubsystem extends DashboardedSubsystem {
     private double currentSpeed = 0;
 
     /**
-     * Constructs a new instance of {@link GenericSubsystem} with the given minimum speed supplier
-     * and maximum speed supplier.
+     * Constructs a new instance of {@link GenericSubsystem} with the given minimum speed supplier,
+     * maximum speed supplier and with the given namespace name.
```

Comment:
namespace name is given first, so it should also be first in the documentation. also add a `@param` for it